Model/ExportInterface.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Hyva\Admin\Model;
+
+use Hyva\Admin\ViewModel\HyvaGridInterface;
+
+interface ExportInterface
+{
+
+    /**
+     * @return string
+     */
+    public function getFileName(): string;
+
+    /**
+     * @param $fileName
+     * @return ExportInterface
+     */
+    public function setFileName($fileName): ExportInterface;
+
+    /**
+     * @return string
+     */
+    public function getMetaType(): string;
+
+    /**
+     * @param $metaType
+     * @return ExportInterface
+     */
+    public function setMetaType($metaType): ExportInterface;
+
+    public function create();
```

Comment:
I've changed it to createFileToDownload so that the developer is able to decide if he wants to get the file (that might be in the file system already) or to create it.